### PR TITLE
More Verbose Output

### PR DESCRIPTION
**Pull request by goruha**
## what
* Added log level configuration with environment variable LOG_LEVEL
* Fixed issue with scheduled jobs panic that crash whole process
* Fixed incidental remove etcd cache when github.com connection failed
* Use `/usr/bin` instead of `/bin`

## who
@cloudposse/engineering

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,6 +434,14 @@ docker build \
   ./
 ```
 
+### Log level
+
+To manage logging level user environment variables ``LOG_LEVEL`` possible values are
```
Comment (osterman) [inline README.md]: "Use the `LOG_LEVEL` environment variable to control the logging verbosity. Possible values are:"

Comment (goruha) [inline README.md]: Done.

File: api/github.go [hunk 2]
```diff
@@ -25,6 +25,14 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var (
+	// ErrorGitHubConnectionFailed - returned when there was connection error github.com
```
Comment (goruha) [inline api/github.go]: Done

**Comment by osterman** [comment]
Sorry to say....

image

This works...

image

**Comment by osterman** [comment]
You can fix the `/opt/bin` path in a separate PR.

**Pull request merged**